man/systemd-vconsole-setup.xml

```diff
@@ -36,30 +36,35 @@
   </refentryinfo>
 
   <refmeta>
-    <refentrytitle>systemd-vconsole-setup.service</refentrytitle>
+    <refentrytitle>systemd-vconsole-setup</refentrytitle>
     <manvolnum>8</manvolnum>
   </refmeta>
 
   <refnamediv>
-    <refname>systemd-vconsole-setup.service</refname>
     <refname>systemd-vconsole-setup</refname>
-    <refpurpose>Configure the virtual console at boot</refpurpose>
+    <refname>systemd-vconsole-setup.service</refname>
+    <refpurpose>Configure the virtual consoles</refpurpose>
   </refnamediv>
 
   <refsynopsisdiv>
+    <cmdsynopsis>
+      <command>systemd-vconsole-setup</command>
+      <arg choice="opt">TTY</arg>
+    </cmdsynopsis>
     <para><filename>systemd-vconsole-setup.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-vconsole-setup</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
     <title>Description</title>
 
-    <para><filename>systemd-vconsole-setup.service</filename> is an
-    early boot service that configures the virtual console font and
-    console keymap. Internally it calls
+    <para><filename>systemd-vconsole-setup</filename> is a helper used to setup either all virtual consoles or a
+      specific one if optional TTY parameter is provided. During bootup it's called by <command>udev</command> during
```

Comment:
Ok, but manual updates aside - does the whole idea of pushing systemd-vconsole-setup to PATH directory make sense ?

Or should I just update man pages and leave it as it is now.